target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -81,6 +81,16 @@ dlink,dwr-960)
 	ucidef_set_led_switch "lan" "lan" "green:lan" "switch0" "0x2e"
 	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x01"
 	;;
+domywifi,dm202|\
+domywifi,dm203|\
+domywifi,dw22d)
+	ucidef_set_led_switch "lan1" "lan1" "amber:lan1" "switch0" "0x02" "" "link"
```

Comment:
Can you comment on what the "" is meant to achieve?

Reply:
https://github.com/openwrt/openwrt/blob/97e820c6d61d174da1091a692546e979a5a45e42/package/base-files/files/lib/functions/uci-defaults.sh#L489
The sixth argument of function is speed mask and we don't need it so I referred to the settings of tplink,archer-d50-v1.
https://github.com/openwrt/openwrt/blob/97e820c6d61d174da1091a692546e979a5a45e42/target/linux/ath79/generic/base-files/etc/board.d/01_leds#L298
"" is just a placeholder here and it seems can work well.

Reply:
Ah, now I get it. This is the first time ever I see this because nobody ever tried to set mode on ucidef_set_led_switch before. (and ucidef_set_led_netdev has different options).

Please just drop the `"" "link"` arguments and set up the LEDs with default mode like all the others, as that's expected default behavior.

Reply:
It has been dropped now.